src/v1/key_management_service_client.ts

```diff
@@ -3415,23 +3416,23 @@ export class KeyManagementServiceClient {
    */
 
   getIamPolicy(
-    request: protos.google.iam.v1.GetIamPolicyRequest,
+    request: IamProtos.google.iam.v1.GetIamPolicyRequest,
     options: gax.CallOptions,
-    callback: protos.google.iam.v1.IAMPolicy.GetIamPolicyCallback
+    callback: IamProtos.google.iam.v1.IAMPolicy.GetIamPolicyCallback
   ) {
     return this._iamClient.getIamPolicy(request, options, callback);
   }
   setIamPolicy(
```

Comment:
I'm pretty sure all IAM methods support calling without a callback. Can we also have the declaration returning promises, as we do for regular simple methods?

Reply:
sure, added at micro-gen side, thank you!